partner-chains-cli/src/prepare_configuration/prepare_main_chain_config.rs

```diff
@@ -129,6 +132,21 @@ fn run_sidechain_main_cli_addresses<C: IOContext>(
 	Ok(())
 }
 
+fn prepare_native_token<C: IOContext>(context: &C) -> anyhow::Result<()> {
+	if context.prompt_yes_no("Do you want to configure a native token for you Partner Chain?", true)
```

Comment:
I'm missing a better explanation here - what is it for and how to prepare the configuration variables.

Reply:
I added a slightly more verbose explanation in the wizard.